stock_picking_batch_creation/models/stock_device_type.py

```diff
@@ -41,6 +41,11 @@ class StockDeviceType(models.Model):
         readonly=False,
     )
     nbr_bins = fields.Integer(string="Number of compartments")
+    split_mode = fields.Selection(
+        selection=[("dimension", "Dimension")],
```

Comment:
In the wizard, in _split_first_picking_for_limit you maybe want to also check this new value ?